pageFactory/pageRepository/SwagLoginPage.ts

```diff
@@ -10,7 +10,9 @@ export class SwagLoginPage {
     readonly USERNAME_EDITBOX: Locator;
     readonly PASSWORD_EDITBOX: Locator;
     readonly LOGIN_BUTTON: Locator;
-    readonly BOOKS_SEARCH_BOX: Locator;
+    readonly error_message: Locator;
```

Comment:
maintain consistency in naming variables, ie., error_message to be ERROR_MESSAGE